src/components/tiger/tiger.tsx

```diff
@@ -0,0 +1,252 @@
+    );
+
+    /**
+     * Handles form submission for the Tiger component.
+     * @param {FormValuesType} values - The form values containing the target domain.
+     */
+    const onSubmit = async (values: FormValuesType) => {
+        // Activate loading state to indicate ongoing process
+        setLoading(true);
+
+        // Construct arguments for the Tiger command based on form input
+        let args = [];
+        args = [values.targetIP, "-l", values.auditLevel];
+
+        // Check if reportFile has a value and push it to args
+        if (values.reportFile) {
+            args.push("-R", values.reportFile);
+        }
+
+        // Check if enableModules has a value and push it to args
+        if (values.enableModules) {
+            args.push("-E", values.enableModules);
+        }
+
+        // Check if excludeModules has a value and push it to args
+        if (values.excludeModules) {
+            args.push("-X", values.excludeModules);
+        }
+
+        if (verboseMode) {
+            args.push("-v"); // Add verbose mode option if enabled
+        }
+
+        // Execute the Tiger command via helper method and handle its output or potential errors
+        CommandHelper.runCommandWithPkexec("tiger", args, handleProcessData, handleProcessTermination)
+            .then(() => {
+                // Deactivate loading state
+                setLoading(false);
+            })
+            .catch((error) => {
+                // Display any errors encountered during command execution
+                setOutput(`Error: ${error.message}`);
+                // Deactivate loading state
+                setLoading(false);
+            });
+        setAllowSave(true);
+    };
+
+    /**
+     * Handles the completion of output saving by updating state variables.
+     */
+    const handleSaveComplete = () => {
+        setHasSaved(true); // Set hasSaved state to true
+        setAllowSave(false); // Disallow further output saving
+    };
+
+    /**
+     * Clears the command output and resets state variables related to output saving.
+     */
+    const clearOutput = () => {
+        setOutput(""); // Clear the command output
+        setHasSaved(false); // Reset hasSaved state
+        setAllowSave(false); // Disallow further output saving
+    };
+
+    // Render component
+    return (
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                />
+            )}
+            <form onSubmit={form.onSubmit(onSubmit)}>
+                <Stack>
+                    {LoadingOverlayAndCancelButton(loading, pid)}
```

Comment:
This should be the loading overlay for pkexec. 
```suggestion
                  {LoadingOverlayAndCancelButtonPkexec(loading, pid, handleProcessData, handleProcessTermination)}
```